Makefile

```diff
@@ -30,15 +30,23 @@ SSE_S := $(shell grep -o sse /proc/cpuinfo | head -n 1)
 ifeq ("$(BUILD_AVX)","1")
 CFLAGS += -mavx -mavx2 -DENABLE_AVX
 else
-ifeq ($(SSE_S),sse)
```

Comment:
I proposed #55 to improve SSE detection for both Linux and macOS. You should be aware of it.